### DISCUSSION TITLE
is it possible to use godot 2.1.6 with the newer version?

### DISCUSSION THREAD

**Discussion by girng** [nigels-com/glew#368] [Q&A] [answered]
i'm using a heavily modified version of 2.1.6 optimized for 2d. was thinking about upgrading its GLEW libray, but i'm not really a graphics programmer, although I have some c/c++ experience. would it be possible/worth it to upgrade to the latest GLEW? afaik, godot 2.1.6 is using the 2007 library.

would there be any performance improvements? (if not, i guess you can close this topic). just curious, thank you!

**Comment by nigels-com** [comment, marked as answer]
I don't think GLEW has changed all that much since 2007.  It has added new versions of OpenGL and extensions over time.
I don't think there would be any particular performance improvements.

**Reply by girng** [reply]
thanks for the swift response, have a great day
